spotbugs-tests/src/test/java/edu/umd/cs/findbugs/detect/Issue2836Test.java

```diff
@@ -1,22 +1,16 @@
 package edu.umd.cs.findbugs.detect;
 
-import static edu.umd.cs.findbugs.test.CountMatcher.containsExactly;
-import static org.hamcrest.MatcherAssert.assertThat;
 
 import org.junit.jupiter.api.Test;
 
 import edu.umd.cs.findbugs.AbstractIntegrationTest;
-import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcher;
-import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcherBuilder;
 
 class Issue2836Test extends AbstractIntegrationTest {
     @Test
     void testIssue() {
         performAnalysis("ghIssues/Issue2836.class");
 
-        BugInstanceMatcher bugTypeMatcher = new BugInstanceMatcherBuilder()
-                .build();
-
-        assertThat(getBugCollection(), containsExactly(0, bugTypeMatcher));
+        assertNoBugType("NP_LOAD_OF_KNOWN_NULL_VALUE");
+        assertNoBugType("RCN_REDUNDANT_NULLCHECK_OF_NULL_VALUE");
```

Comment:
Original issue (https://github.com/spotbugs/spotbugs/issues/2836) is about NP_LOAD_OF_KNOWN_NULL_VALUE and RCN_REDUNDANT_NULLCHECK_OF_NULL_VALUE FPs.